alws/models.py

```diff
@@ -254,11 +254,13 @@ class Platform(PermissionsMixin, Base):
     priority: Mapped[Optional[str]] = mapped_column(
         sqlalchemy.Integer, nullable=True
     )
-    arch_list: Mapped[Dict[str, Any]] = mapped_column(JSONB, nullable=False)
-    copy_priority_arches: Mapped[Optional[Dict[str, Any]]] = mapped_column(
-        JSONB, nullable=True
+    arch_list: Mapped[List[Dict[str, Any]]] = mapped_column(
+        JSONB, nullable=False
+    )
+    copy_priority_arches: Mapped[List[Optional[Dict[str, Any]]]] = (
+        mapped_column(JSONB, nullable=True)
     )
-    weak_arch_list: Mapped[Optional[Dict[str, any]]] = mapped_column(
+    weak_arch_list: Mapped[List[Optional[Dict[str, any]]]] = mapped_column(
```

Comment:
I see that you replace `Mapped[Optional[Dict` as `Mapped[Optional[List[Dict` all over the place except here, why `Mapped[List[Optional[Dict` here? typo?

Reply:
Yep, it wasn't intended. Thank you for pointing out!